simple-platform/src/main/java/org/simple/clinic/platform/crash/Breadcrumb.kt

```diff
@@ -1,4 +1,4 @@
-package org.simple.clinic.crash
+package org.simple.clinic.platform.crash
```

Comment:
Nice! I was planning to do this, but looks like you already have.

There is a difference between what you have done, and what I was planning though. My plan was to make this a pure java module. We can do this by removing the `init` function from the interface and moving the initialisation to the constructor of `SentryCrashReporter`.

We can do this in a later PR, though.